Libplanet/Net/Swarm.cs

```diff
@@ -182,7 +181,11 @@ static Swarm()
 
         public Peer AsPeer => Transport.AsPeer;
 
-        public DateTimeOffset LastReceived { get; private set; }
+        /// <summary>
+        /// The time the message was last received.
+        /// </summary>
```

Comment:
```suggestion
```suggestion
        /// <summary>
        /// The last time when any message was arrived.
        /// It can be <c>null</c> if no message has been arrived yet.
        /// </summary>
        public DateTimeOffset LastMessageTimestamp { get; private set; }
```
```